test/extension/commandExecutor.test.ts

```diff
@@ -125,5 +127,70 @@ suite("commandExecutor", function() {
                     assert.fail(null, null, "react-native command was not expected to fail");
                 });
         });
+
+        suite("getReactNativeVersion", function () {
```

Comment:
Since we implemented version checking functions in a separate class, I suppose we should also implement the tests for it in a separate file.

Reply:
Unit tests for `ReactNativeProjectHelper` class was created in `reactNativeProjectHelper.test` file.